### ISSUE TITLE
GraphViz2 2.51: test t/gen.Heawood.t fails on darwin19 (macOS 10.15 Catalina)

### ISSUE THREAD

**Issue by dbevans** [ronsavage/GraphViz2#7]
Using:

```
macOS 10.15.7 19H2
Xcode 12.0.1 12A7300 
MacPorts perl 5.30.3
```
```
#   Failed test 'dot file'
#   at t/gen.Heawood.t line 72.
# @@ -1,7 +1,7 @@
#  'graph Heawood {
#   graph [ rankdir="TB" ]
#   edge [ color="black" ]
# - node [ color="black" fontname="Arial" height="0.500000" label="\\N" shape="circle" width="0.50000" ]
# + node [ color="black" fontname="ArialMT" height="0.500000" label="\\N" shape="circle" width="0.50000" ]
#   "0"
#   "1"
#   "0" -- "1"
# Looks like you failed 1 test of 1.
t/gen.Heawood.t ............ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 

```

**Comment by dbevans** [comment]
Preinstalled Arial fonts on Catalina are (see https://support.apple.com/en-us/HT210192)

```
Arial
Arial Black
Arial Hebrew
Arial Hebrew Scholar
Arial Narrow
Arial Rounded MT Bold
Arial Unicode MS

```

**Comment by ronsavage** [comment]
Hi David

Thanx for the report.

I must have a long time ago run into this problem, because there is 
special code in that test program for Darwin:

t/gen.Heawood.t
41: fontname => ($^O eq 'darwin') ? "ArialMT" : "Arial",

I'm including Ed in the reply since he's doing a lot of work on 
GraphViz2 ATM. We'll see what he thinks...

---
Cheers
Ron
savage.net.au

**Comment by mohawk2** [comment]
Thanks for the report! I've zapped the special case (I did notice it while converting the test, but assumed it was still correct) and released 2.52. Please try it once it hits CPAN (within about an hour of now) and reopen this if it's still a problem :-)

**Comment by dbevans** [comment]
> Thanks for the report! I've zapped the special case (I did notice it while converting the test, but assumed it was still correct) and released 2.52. Please try it once it hits CPAN (within about an hour of now) and reopen this if it's still a problem :-)

Looks good.  Updated to 2.54 this morning with all tests passing.  Will push the update out to MacPorts later today. Thanks for the quick response.

**Comment by mohawk2** [comment]
Welcome!
